tests/integration/message/message.spec.ts

```diff
@@ -2,345 +2,212 @@ import jsLogger from '@map-colonies/js-logger';
-        useChild: true,
-      });
-
-      requestSender = await createRequestSender<paths, operations>('openapi3.yaml', app);
-      localMessagesStore.length = 0;
-
-      console.log('✅ App and requestSender initialized');
-
-      const repo = messageLogsDataSource.getRepository(Message);
-      console.log('🛠 Repository:', repo.metadata.tableName);
-
-      await repo.clear();
-      console.log('✅ Message table cleared');
-    } catch (err) {
-      console.error('❌ Error in beforeEach:', err);
-      throw new Error('💥 Failing in beforeEach: ');
-    }
-  });
+  requestSender = await createRequestSender<paths, operations>('openapi3.yaml', app);
+});
 
-  describe('Happy Path', function () {
-    it('should return 201 status code and the internal id', async function () {
-      const response = await requestSender.createMessage({
-        requestBody: fullMessageInstance,
-      });
+beforeEach(() => {
+  localMessagesStore.length = 0;
+  jest.clearAllMocks();
+});
 
-      expect(response).toSatisfyApiSpec();
-      expect(response.status).toBe(httpStatusCodes.CREATED);
-    });
+// -------------------- Happy Path --------------------
+describe('Message Integration Tests - Happy Path', () => {
+  it('should return 201 when creating a message', async () => {
+    const response = await requestSender.createMessage({ requestBody: fullMessageInstance });
+    expect(response).toSatisfyApiSpec();
+    expect(response.status).toBe(httpStatusCodes.CREATED);
+  });
 
-    it('should return 200 status code and appropriate message when no messages match filters', async function () {
-      const messageInstance: DeepPartial<Message> = {
-        id: uuidv4(),
-        sessionId: '9999',
-        severity: 'ALERT' as SeverityLevels,
-        component: 'GENERAL' as LogComponent,
-        messageType: 'APPSTARTED' as AnalyticsMessageTypes,
-        message: 'Non-matching message',
-        timeStamp: new Date(),
-      };
-
-      await messageLogsDataSource.getRepository(Message).save(messageInstance);
-
-      const response = await requestSender.getMessages({
-        queryParams: {
-          sessionId: '22342',
-          severity: 'ERROR',
-          component: 'MAP',
-          messageType: 'APPEXITED',
-        },
-      });
-
-      expect(response).toSatisfyApiSpec();
-      expect(response.status).toBe(httpStatusCodes.OK);
-      expect(response.body).toEqual([]);
-    });
+  it('should return all messages if no query params', async () => {
+    const response = await requestSender.getMessages();
+    expect(response).toSatisfyApiSpec();
+    expect(response.status).toBe(httpStatusCodes.OK);
+    expect(response.body).toHaveLength(1);
+  });
 
-    it('should return 200 status code and handling if no query params provided', async function () {
-      const messageInstance: DeepPartial<Message> = {
-        id: uuidv4(),
+  it('should return filtered messages', async () => {
+    const response = await requestSender.getMessages({
```

Comment:
* Get and check that none exists
* Add 2 messages
* Get and check that filtered exists